src/xdocs/checks/annotation/annotationusestyle.xml

```diff
@@ -130,116 +130,139 @@
         </div>
       </subsection>
       <subsection name="Examples" id="Examples">
-        <p> To configure the check:</p>
+        <p id="Example1-config">
+          To configure the check:
+        </p>
         <source>
-&lt;module name=&quot;AnnotationUseStyle&quot;/&gt;
+&lt;module name=&quot;Checker&quot;&gt;
+  &lt;module name=&quot;TreeWalker&quot;&gt;
+    &lt;module name=&quot;AnnotationUseStyle&quot;/&gt;
+  &lt;/module&gt;
+&lt;/module&gt;
         </source>
-        <p>
-          Example:
-        </p>
+        <p id="Example1-code">Example:</p>
         <source>
-@SuppressWarnings(&quot;unchecked&quot;) // OK
-@Deprecated // OK
-@SomeArrays({&quot;unchecked&quot;,&quot;unused&quot;}) // OK
-public class TestOne
+@SuppressWarnings(&quot;unchecked&quot;) // ok as it's in implied style
+@Deprecated // // ok as it matches closingParens default property
+@SomeArraysUnique({&quot;unchecked&quot;,&quot;unused&quot;}) // ok as it's in implied style
```

Comment:
What forcing you to change annotation name to SomeArraysUnique ?
If some tests Inputs declare them already, let's change them.
We need to keep examples to be compact because all users will read them and java Input that we use  for just testing almost nobody reads.

Reply:
Duplicates issue. Done. Please let me know your feedback.